### DISCUSSION TITLE
Can't import teleport certificate

### DISCUSSION THREAD

**Discussion by twf0** [gravitational/teleport#40292] [Q&A] [answered]
Hi,

I'm trying to setup Windows Desktop on a windows 10 VM (kvm), but i have this error when i launch the setup.exe
![Capture d'écran 2024-04-06 183730](https://github.com/gravitational/teleport/assets/99323354/9544b5dd-02cb-457f-9915-cf358287f17e)

It's working on another windows 10 (but not in KVM), does someone have an idea ? 

Thanks !

**Comment by zmb3** [comment, marked as answer]
The certificate file is likely invalid or corrupt. This can happen if you put an extra trailing `/` in the curl command.

I would export with `tctl auth export --type=windows` and see if you get the same file.

**Reply by webvictim** [reply]
I've seen this exact same issue come up quite a few times; raised https://github.com/gravitational/teleport/pull/40352 to address it.
